src/routes/safe/store/actions/transactions/utils/transactionHelpers.ts

```diff
@@ -340,7 +340,7 @@ export const mockTransaction = (tx: TxToMock, safeAddress: string, state): Promi
     ...tx,
   }
 
-  const knownTokens: Record<string, Token> = state[TOKEN_REDUCER_ID]
+  const knownTokens: Map<string, Token> = state[TOKEN_REDUCER_ID]
```

Comment:
I would extract this type to a separate one like `TokensReduxState` and place it inside the reducer. So, whenever we change the reducer, we easily change the type and all the consumers get the new type too